index.ts

```diff
@@ -1,17 +1,19 @@
 import { createBuffer } from '@posthog/plugin-contrib'
 import { Plugin, PluginMeta, PluginEvent } from '@posthog/plugin-scaffold'
-import { BigQuery, Table } from '@google-cloud/bigquery'
+import { BigQuery, Table, TableField, TableMetadata } from '@google-cloud/bigquery'
 
 class RetryError extends Error {}
```

Comment:
Outside of the changes of the PR, but let's import this from '@posthog/plugin-scaffold'